docs/glossary.md

```diff
@@ -0,0 +1,86 @@
+# Glossary
+
+- [API Review](#api-review)
+- [Code Freeze](#code-freeze)
+- [Exceptions](#exceptions)
+- [Enhancements Freeze](#enhancements-freeze)
+- [Kubernetes Enhancements Proposal (KEP)](#kubernetes-enhancement-proposal-kep)
+- [Production Readiness Review (PRR)](#production-readiness-review-prr)
+
+### API Review
+The API review process is intended to maintain logical and functional integrity of the
+ API over time, the consistency of user experience and the ability of previously
+ written tools to function with new APIs.
+
+If an enhancement is considered to be changing the Kubernetes core API in any way,
+ a mandatory API review is required. If you are unsure, please do have a look at
+ the document linked below. An Enhancement owner should proactively request an
+ API review for an enhancements PR by adding a `/label api-review` comment.
+ The status of an API review can be found in the
+ [API review backlog](https://github.com/orgs/kubernetes/projects/13).
+
+More details can be found in [Kubernetes API Review Process](https://github.com/kubernetes/community/blob/master/sig-architecture/api-review-process.md).
+
+### Code Freeze
+All enhancements going into the release must be code-complete by the code freeze deadline.
+ Any unmerged pull requests in k/k repo for the upcoming release milestone will be removed
+ by the Release Team and will require an approved exception to have the milestone added back
+ by the Release Team.
+
+More details can be found in [Code Freeze](https://github.com/kubernetes/sig-release/blob/master/releases/release_phases.md#code-freeze).
+
+### Exceptions
+When an enhancement fails to meet the release deadline, Enhancement owner must file an
+ [exception request](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md#requesting-an-exception)
+ addressed to SIG sponsoring the enhancement, the Release Team, and SIG Release to gain approval.
+ The release team will be responsible for approving or rejecting exceptions and the decision
+ will be based on discussion with the sponsoring SIG's chairs/leads and any other participating
+ SIG's chairs/leads.
+
+More details can be found in [Exceptions to Milestone Enhancement Complete dates](https://github.com/kubernetes/sig-release/blob/master/releases/EXCEPTIONS.md).
+
+### Enhancements Freeze
+All enhancements going into the release must meet the following criteria by the enhancements freeze deadline:
+- An open issue in the k/enhancements repo
```

Comment:
would prefer to not duplicate content as it tends to go out of date.  we can just link to the sig-release criteria here and let reader go there to see most current version.